linode/manager
# feat: [M3-7025] - AGLB Details - Certificates Tab

## Description 📝
- Adds the AGLB certificate tab to the AGLB details page 🎉
  - What I built out is similar to the mockup but I left out some table columns that would require a lot of client-side computation for now

## Preview 📷
![Screenshot 2023-08-21 at 8 30 45 PM](https://github.com/linode/manager/assets/115251059/9362d0fa-002c-4c2f-9c89-66560c27c11d)


## How to test 🧪
- Turn on the MSW
- Navigate to `http://localhost:3000/loadbalancers/1/certificates`
- Test basic functionality
  - Note that sorting and filtering won't work because the MSW does not handle API filters
